src/ng-generate/components/table/schema.ts

```diff
@@ -12,9 +12,18 @@
  */
 
 import {Values} from '../shared/schema';
+import data from './schema.json';
 
 export interface TableSchema extends Values {
     customRowActions: string[];
     addRowCheckboxes: boolean;
     customColumns: Array<string>;
 }
+
+export interface ITableDefaultsSchema {
```

Comment:
Using "I" as a prefix for an interface is an anti-pattern, see also the official Angular style guide.

Reply:
done